packages/plugins/Approval/src/SNSAdaptor/ApprovalTokenContent.tsx

```diff
@@ -203,7 +203,7 @@ function ApprovalTokenItem(props: ApprovalTokenItemProps) {
         chainId,
     )
 
-    const { value: token } = useFungibleToken(NetworkPluginID.PLUGIN_EVM, spender.tokenInfo.address, undefined, {
+    const { data: token } = useFungibleToken(NetworkPluginID.PLUGIN_EVM, spender.tokenInfo.address, undefined, {
```

Comment:
I found some plugins are not listed in `packages/plugins/tsconfig.json`. After adding that, here comes type error. @UncleBill is this fix correct?

Reply:
Yes.